skaraMirror.sh

```diff
@@ -269,7 +269,7 @@ fi
 jdk11plus_tag_sort1="sort -t+ -k2,2n"
 # Second, (stable) sort on (V), (W), (X), (P): P(Patch) is optional and defaulted to "0"
 jdk11plus_tag_sort2="sort -t. -k2,2n -k3,3n -k4,4n -k5,5n"
-jdk11plus_sort_tags_cmd="grep -v _adoptium | sed 's/jdk-/jdk./g' | sed 's/+/.0.0+/g' | $jdk11plus_tag_sort1 | nl -n rz | $jdk11plus_tag_sort2 | sed 's/\.0\.0+/+/g' | cut -f2- | sed 's/jdk./jdk-/g'"
+jdk11plus_sort_tags_cmd="egrep -v '(_adopt|_adoptium)'  | sed 's/jdk-/jdk./g' | sed 's/+/.0.0+/g' | $jdk11plus_tag_sort1 | nl -n rz | $jdk11plus_tag_sort2 | sed 's/\.0\.0+/+/g' | cut -f2- | sed 's/jdk./jdk-/g'"
```

Comment:
SInce we're not breaking on a word binary, is there any reason not to leave it as `grep -v _adopt`?

Reply:
this PR  is need if we still want to have both _adopt and _adoptium tags, 
but if we go with the solution of revert back to only use _adopt, this PR is not needed

Reply:
I get that, but `grep -v _adopt` is simpler and would cover both cases.

Reply:
ah, now i get it! 
yes, _adopt covers _adoptium => one "_adopt" is enough in this case.